### PR TITLE
[Term Entry] Go string function contains

### PR DESCRIPTION
**Pull request by Manikanta528**
### Description

Addressing issue #2631 

### Type of Change

- Adding a new entry Go string function contains

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/go/concepts/strings/terms/contains/contains.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+The **`Contains()`** function returns a boolean value indicating whether a given substring is present or not in a given string. If given substring is present in the given string then it returns true, else it returns false.
+
+## Syntax
+
+```pseudo
+strings.Contains(str, sub_str)
+```
+
+Where `str` is the original string and `sub_str` is the substring that we wanted to check against the original string.
```
Comment (SSwiniarski) [inline content/go/concepts/strings/terms/contains/contains.md:23]:
```suggestion
Where `str` is the original string and `sub_str` is the substring to check against the original string.
```
Avoid first- and second-person pronouns.

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@Manikanta528 I did a first review and had one comment.

**Comment by Manikanta528** [comment]
@SSwiniarski updated the requested changes

**Review by SSwiniarski — APPROVED** [review]
@Manikanta528 looks good for a second review.

**Review by caupolicandiaz — APPROVED** [review]
@Manikanta528 @SSwiniarski I just made a couple of minor edits. Looks good to merge

**Comment by SSwiniarski** [comment]
@Manikanta528 congrats on the new entry. Live link here: https://www.codecademy.com/resources/docs/go/strings/contains

image

**Pull request merged**